docs/endpoints/installation.rst

```diff
@@ -0,0 +1,123 @@
+Installing the Compute Endpoint
+*******************************
+
+The Globus Compute Endpoint is offered both as a PyPI package, and as pre-built DEB and
+RPM packages.
+
+Operating System Support
+========================
+
+Currently, the Globus Compute Endpoint is only supported on Linux.  While some have
+reported success running Compute endpoints on macOS, we do not currently support
+it.  If running on a non-Linux host OS is necessary, consider doing so in a container
+running Linux.
+
+.. note::
+
+   Though the Compute Endpoint is only supported on Linux, the
+   :doc:`Globus Compute SDK <../sdk>` *is* supported on other operating systems.
+
+.. _pypi-based-installation:
+
+Installing Directly via PyPI
+============================
+
+If the site administrator has not already installed the Globus Compute Endpoint
+software, users typically install it from `PyPI
+<https://pypi.org/project/globus-compute-endpoint/>`_.  We **strongly** recommend
+installing the Globus Compute endpoint into an isolated virtual environment (e.g.,
+`venv <https://docs.python.org/3/library/venv.html>`_, `virtualenv
+<https://pypi.org/project/virtualenv/>`_.  We
+recommend use of |pipx for library isolation|_:
+
+.. code-block:: console
+
+   $ python3 -m pipx install globus-compute-endpoint
+
+
+.. _repo-based-installation:
+
+Repository-Based Installation
+=============================
+
+The ``globus-compute-endpoint`` project is available on PyPI, and is also available in
+Globus' repositories as native DEB and RPM packages.  The repository package is
+``globus-compute-agent``.
+
+.. _compute-endpoint-pre-requisites:
+
+Prerequisites
+-------------
+
+#. Supported Linux Distributions
+
+   Where feasible, Globus Compute supports the `same Linux distributions as does Globus
+   Connect Server`_.
+
+#. Administrator Privileges
+
+   Per usual semantics, installing the DEB or RPM packages will require administrative
+   access on the target host.
+
+#. TCP Ports
+
+   * Port 443, outbound to ``compute.api.globus.org``
+   * Port 443, outbound to ``compute.amqps.globus.org``
+
+   .. note::
+       We do not offer a range of specific IP addresses for firewall blocking rules.
+
+What is Installed
+-----------------
+
+The package manager packages rely on Globus' supplied Python.  As of this writing, that
```

Comment:
Should this section be moved after 'The Globus Compute Endpoint software will be installed in...'?   ie. the main thing, then the other things it needs.